repository/csv.go

```diff
@@ -0,0 +1,65 @@
+package repository
+
+import (
+	"encoding/csv"
+	"errors"
+	"os"
+	"strconv"
+
+	"github.com/JoseLuisTorrentera/academy-go-q42021/models"
+)
+
+func GetAllSpells() ([]*models.Spell, error) {
```

Comment:
By convention in every exported stuff like (functions, variables, etc) use a single line comment for example:

// GetAllSpells - Description here...
func GetAllSpells() ([]*models.Spell, error) {